configuration/mysql/garage_port_drayage/README.md

```diff
@@ -0,0 +1,11 @@
+# C1T Garage Actions
+These are Port Drayage actions created for the Saxton garage for testing of C1T functionality.
```

Comment:
Update to "These are Port Drayage actions created the CDA1Tenth functionality and specifically tailored for the Turner Fairbank Highway Research Center's Saxton Laboratory garage demonstration."